src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -21,8 +21,8 @@
     private CartMapper cartMapper;
 
     @PostMapping
-    public Cart createEmptyCart() {
-        return new Cart();
+    public void createEmptyCart() {
+        cartService.save(new Cart());
```

Comment:
to jest finalna implementacja? zerknij jak to jest zrobione tutaj: https://github.com/metho/project-JDP-1911-02/blob/08085c707cc9871ca2097ede08828e8bbf661eb0/src/main/java/com/kodilla/ecommercee/controller/OrderController.java#L36